augur/api/view/api.py

```diff
@@ -1,6 +1,7 @@
 from flask import Flask, render_template, render_template_string, request, abort, jsonify, redirect, url_for, session, flash
 from flask_login import current_user, login_required
 from augur.application.db.models import Repo
+from augur.tasks.frontend import add_org_repo_list
```

Comment:
There is no augur.tasks.frontend file or directory @IsaacMilarky / @ABrain7710  .... so boot is failing.